serverless/api/state.go

```diff
@@ -15,6 +15,17 @@
 // Package api contains the "public" API/artifacts of the serverless log.
 package api
 
+import (
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// CheckpointHeader is the first line of a marshaled log checkpoint.
+const CheckpointHeader = "Log Checkpoint v0"
```

Comment:
Stick v0 in the const name (or make it private/lowercase)?

Reply:
done